src/app/routes/index.test.jsx

```diff
@@ -42,6 +43,10 @@ global.performance.getEntriesByName = jest.fn(() => []);
 // mock pages/index.js to return a non async page component
 jest.mock('../pages');
 
+const agent = { ca: 'ca', key: 'key' };
+const getAgent = jest.fn(() => agent);
+const fetchDataSpy = jest.spyOn(fetchPageData, 'default');
```

Comment:
Not sure I understand this. If this makes me unqualified then disregard my CR

Reply:
Haha its all good!

The `agent` stuff is basically mocking up the return values from the `getAgent` function in our data fetcher. `getAgent` allows us to make use of the certificates configured on the server to authenticate our requests to FABL. Previously, the `.json` endpoints were open to the public, whereas FABL is behind a cert.

The `fetchDataSpy` uses Jests `spyOn` function to create a mock function of `fetchPageData`. This lets us then mock the return responses from `fetchPageData`, as seen in your comment below. The `'default'` part of the `spyOn` is just to tell Jest that `fetchPageData` is the default export.